docs/future-work/sensor-module-improvements/detect-local-and-global-flow.md

```diff
@@ -1,3 +1,9 @@
 ---
 title: Detect Local and Global Flow
 ---
+
+Our general view is that there are two sources of flow processed by cortical columns. A larger receptive field sensor helps to estimate global flow, where flow here will be particularly pronounced if the whole object is moving, or the sensor itself is moving. A small receptive-field sensor patch corresponds to the channel by which the primary sensory features (e.g. point-normal, color) arrive. If flow is detected here, but not in the more global channel, then it is likely that just part of the object is moving.
```

Comment:
I would make the distinction more clear:
local flow - object is moving
global flow - sensor is moving

We should also mention that these may not be detectable with the same sensor (small patch can't distinguish between object and sensor movement since for the patch both of it would be global flow.

Reply:
Ok yeah that's what I was trying to get at (the uncertainty depending on the size), will try rewording it.